.github/workflows/testing.yml

```diff
@@ -106,9 +106,12 @@ jobs:
         run: |
           go version
           go test ${{ matrix.testflags }} -cpu 1,4 -timeout 7m google.golang.org/grpc/...
-          cd ${GITHUB_WORKSPACE}/security/advancedtls && go test ${{ matrix.testflags }} -timeout 2m google.golang.org/grpc/security/advancedtls/...
-          cd ${GITHUB_WORKSPACE}/security/authorization && go test ${{ matrix.testflags }} -timeout 2m google.golang.org/grpc/security/authorization/...
-
+          cd "${GITHUB_WORKSPACE}"
+          for MOD_FILE in $(find . -name 'go.mod' | grep -Ev '^\./go\.mod'); do
```

Comment:
This will run the top level tests again.
Limit the `find`?
`find . -mindepth 2 -name 'go.mod'`

Reply:
In the suggested snippet, the second part is `grep -Ev '^\./go\.mod'`, it excludes the top level tests. Can you double check if current test set works as expected? https://github.com/grpc/grpc-go/runs/5994752478?check_suite_focus=true

(It looks reasonable to me, no duplication found)

Reply:
Oh, I see. I somehow thought that was to remove the go.mod and keep only the directory name. This should be good then!